docs/builds.rst

```diff
@@ -60,6 +63,18 @@ key to ``true`` in order for simexpal to pull the respective files.
 
 If ``recursive-clone`` is not set, it will default to ``false``.
 
+Specifying Local Builds
+-----------------------
+
+Specifying a local builds work similar to :ref:`SpecifyingGitRepository`. The differences are
+
+- we omit the ``git`` key *and*
+- store the local build files in ``./develop/<build_name>@<revision_name>``.
+
+(As of now local builds are only supported for :ref:`DevRevisions`)
```

Comment:
Drop the parenthesis, this is important. Maybe even make it bold. Explain the rationale.

Reply:
Done => **As of now local builds are only supported for** Develop Revisions(as link) **as we can not guarantee reproducibility
without some kind of identifier, e.g, a commit hash.**